### PR TITLE
Swaps the variable field label element for the variable name element, ma...

### PR DESCRIPTION
**Pull request by isangil**
...king it more consistent with Drupal and EML conventions.

Low level testing at http://sev-d.lternet.edu, except for the "data explorer" changes.  EML and form look good, perhaps the weight of the variable label element can be changed to effect the order

### CODE REVIEW

File: modules/custom/deims_variable/deims_variable.field.inc [hunk 1]
```diff
@@ -448,7 +448,7 @@ function deims_variable_remove_submit($form, &$form_state) {
 
   // Go one level up in the form, to the widgets container.
   $parent_element = drupal_array_get_nested_value($form, $address);
-  $field_name = $parent_element['#field_name'];
+  $field_name = $parent_element['#field_name'];  //label x name not swapped
```
Comment (davereid) [inline modules/custom/deims_variable/deims_variable.field.inc]: Ok to remove the comment about this line, it has nothing to do with name vs label.

File: modules/custom/deims_data_explorer/deims_data_explorer.module [hunk 2]
```diff
@@ -259,8 +259,8 @@ function deims_data_explorer_form($form, &$form_state, $variables, $connection,
   );
 
   $header  = array(
-    'label' => t('Label'),
-    'name' => t('Variable Name'),
+    'label' => t('Name'),
+    'name' => t('Variable Label'),
```
Comment (davereid) [inline modules/custom/deims_data_explorer/deims_data_explorer.module]: I think we'll want to switch the keys here as well.

Comment (isangil) [inline modules/custom/deims_data_explorer/deims_data_explorer.module]: I think you are saying in the options array -- the array that is set right after $header
something like from this:

 'label' => $variable['label'],
 'name' => $variable['name'],

to

 'name' => $variable['label'],
  'label' => $variable['name'],

but i may be off, im a bit lost of where i was here.

**Comment by davereid** [comment]
Merged and committed with https://github.com/palantirnet/deims-profile/commit/8753c9b1b70b98af6775950dfff02bac678ef742. Thanks Inigo for your hard work on this.

The few major changes that needed to be made in addition to your changes were:
- deims_variable_field_validate() to ensure the name field is never left empty if other values are filled out
- deims_variable.field.js to ensure that the name field is used in the summary
- deims_variable_search_index_variables() to prioritize the 'name' column as higher priority than the 'label' field.

**Pull request closed without merging**